demisto_sdk/commands/download/downloader.py

```diff
@@ -579,6 +587,30 @@ def handle_regex_flag(self) -> None:
                     input_files_regex_match.append(input_file)
             self.input_files = input_files_regex_match
 
+    def handle_init_flag(self) -> None:
+        """
+        Handles the case where the init flag is given
+        :return: None
+        """
+        if not self.init:
+            return
+
+        root_folder = Path(self.output_pack_path) / "Packs"
+        root_folder.mkdir(parents=True, exist_ok=True)
+        initiator = Initiator(str(root_folder))
+        initiator.init()
+        self.output_pack_path = initiator.full_output_path
+        self.all_custom_content = True
+        self.run_format = True
+
+        if self.keep_empty_folders:
+            return
```

Comment:
avoid this return, please change to `if not self.keep...:` and indent the following code, to prevent bugs in case we add some code below in the future